from the aarch64-darwin builder:

```
____________________________ test_profile_captured _____________________________
[gw0] darwin -- Python 3.12.4 /nix/store/3swy1vadi125g0c1vxqp8ykdr749803j-python3-3.12.4/bin/python3.12
sentry_init = <function sentry_init.<locals>.inner at 0x105c23f60>
capture_envelopes = <function capture_envelopes.<locals>.inner at 0x105c8c040>
teardown_profiling = None
    @pytest.mark.forked
    def test_profile_captured(
        sentry_init,
        capture_envelopes,
        teardown_profiling,
    ):
        sentry_init(
            traces_sample_rate=1.0,
            _experiments={"profiles_sample_rate": 1.0},
        )
    
        envelopes = capture_envelopes()
    
        with start_transaction(name="profiling"):
            time.sleep(0.05)
    
        items = defaultdict(list)
        for envelope in envelopes:
            for item in envelope.items:
                items[item.type].append(item)
    
        assert len(items["transaction"]) == 1
>       assert len(items["profile"]) == 1
E       assert 0 == 1
E        +  where 0 = len([])
```

Seems to be timing related, i think the test can be disabled as "flaky"

--- a/pkgs/development/python-modules/sentry-sdk/default.nix
+++ b/pkgs/development/python-modules/sentry-sdk/default.nix
@@ -187,6 +187,8 @@ buildPythonPackage rec {
     # assert count_item_types["sessions"] == 1
     # assert 0 == 1
     "test_auto_session_tracking_with_aggregates"
+    # timing sensitive
+    "test_profile_captured"
   ];
 
   pythonImportsCheck = [ "sentry_sdk" ];

